dapp/src/components/Nav.js

```diff
@@ -34,9 +34,32 @@ const DappLinks = ({ dapp, page }) => {
 
   return (
     <>
+      {!dapp && (
```

Comment:
are there any situations where the `dapp === false`?

Reply:
I can merge the PR in and this can be addressed later if needed. Doesn't seem it could do any damage